CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0-32
+
+- Add japanese traffic signs
```

Comment:
nit:

```suggestion
- Added japanese traffic signs
```

as we use past simple earlier?

Reply:
Right, thanks.